dom.bs

```diff
@@ -1075,11 +1084,17 @@ steps:
 
  <li><p>Let <var>once</var> and <var>passive</var> be false.
 
+ <li><p>Let <var>signal</var> be null.
+
  <li><p>If <var>options</var> is a dictionary, then set <var>passive</var> to <var>options</var>'s
- <code>{{AddEventListenerOptions/passive}}</code> and <var>once</var> to <var>options</var>'s
+ <code>{{AddEventListenerOptions/passive}}</code>, <var>once</var> to <var>options</var>'s
```

Comment:
Please restore "and" here. Comma isn't needed then.

Reply:
You might want to clean this up while here to use the same accessing syntax as below. So instead of `options's x` use `options[x]`.

Reply:
I'm not entirely sure how to clean up syntax wise? I'll restore the `and`